### ISSUE TITLE
Doesn't open external terminal at right folder.

### ISSUE THREAD

**Issue by RicaBenhossi** [microsoft/vscode#131825]
Issue Type: <b>Bug</b>

When you selecta folder on explorer, open context menu and select option  "OPEN IN EXTERNAL TERMINAL" it opens the exteranal terminal, but not in the folder you clieck in. It opens on home/user.

VS Code version: Code - Insiders 1.60.0-insider (6a2daec767703dad42461333fbf8666aa481a32b, 2021-08-27T08:45:52.403Z)
OS version: Linux x64 5.11.0-27-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-3110M CPU @ 2.40GHz (4 x 2318)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|7.67GB (0.31GB free)|
|Process Argv|--unity-launch --crash-reporter-id 439cf94a-b7d3-4d49-a4ca-5fa78c9cd20b|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|pantheon|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|pantheon|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (62)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
vscode-tomcat|ada|0.11.3
Bookmarks|ale|13.1.0
project-manager|ale|12.3.0
read-only-indicator|ale|3.5.0
arepl|alm|2.0.3
gitkraken-glo|axo|1.1.1
vscode-django|bat|1.6.0
gitignore|cod|0.7.0
bracket-pair-colorizer|Coe|1.0.61
doxdocgen|csc|1.3.2
python-preview|don|0.0.4
githistory|don|0.6.18
python-extension-pack|don|1.6.0
xml|Dot|2.5.1
gitlens-insiders|eam|2021.7.1905
norminette-42|evi|1.0.8
vscode-firefox-debug|fir|2.9.4
pythonsnippets|frh|1.0.2
codespaces|Git|1.0.3
vscode-pull-request-github|Git|0.29.2
todo-tree|Gru|0.0.213
selected-lines-count|gur|1.4.0
vscode-edit-csv|jan|0.6.0
better-cpp-syntax|jef|1.15.6
42header|kub|0.42.9
rainbow-csv|mec|1.9.1
vscode-docker|ms-|1.16.0
python|ms-|2021.8.1159798656
vscode-pylance|ms-|2021.8.3
jupyter|ms-|2021.8.2031173646
remote-containers|ms-|0.194.0
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
remote-ssh-edit-nightly|ms-|2021.8.38518
remote-ssh-nightly|ms-|2021.8.38518
remote-wsl|ms-|0.58.2
vscode-remote-extensionpack|ms-|0.21.0
cpptools|ms-|1.6.0
cpptools-extension-pack|ms-|1.0.0
vsliveshare|ms-|1.0.4761
vsliveshare-audio|ms-|0.1.91
debugger-for-chrome|msj|4.13.0
resourcemonitor|mut|1.0.7
embrace|myc|1.0.0
autodocstring|njp|0.5.4
java|red|0.81.0
vscode-open|san|0.1.0
snipsnap-vscode|sni|1.1.1
remove-empty-lines|use|0.0.8
vscodeintellicode|Vis|1.2.14
vscode-java-debug|vsc|0.35.0
vscode-java-dependency|vsc|0.18.7
vscode-java-pack|vsc|0.18.4
vscode-java-test|vsc|0.31.3
vscode-maven|vsc|0.33.0
vscode-icons|vsc|11.6.0
vscode-wakatime|Wak|15.0.2
jinja|who|0.0.8
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.8.0
vscode-open-in-github|ziy|1.3.6

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30291494
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30291487
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
pythonvsuse255:30319630
vscod805cf:30301675
pythonvspyt200:30323110
vsccppwt:30312692
pythonvssor306:30340298
bridge0708:30335490
pygetstartedt2:30353727
dockerwalkthrucf:30348714
bridge0723:30353136
pythonrunftest32:30353181
pythonf5test824cf:30353184
javagetstartedt:30350119
pythonvspyt187cf:30356035
pydsgst2:30356659

```

</details>

**Comment by meganrogge** [comment]
Please provide a gif of the issue

**Comment by RicaBenhossi** [comment]
Sure....
![image](https://user-images.githubusercontent.com/15204313/131900506-2e742650-8330-4f51-8741-978397d9cfee.png)

**Comment by meganrogge** [comment]
thanks for the photo. can you pls provide a gif ? https://gifcap.dev/

**Comment by RicaBenhossi** [comment]
Fudge..... 
Sorry, my bad    \o

Don't know why it did converted gif to png.

Here is the real gif. Sorry again.....

![vscode-external-terminal](https://user-images.githubusercontent.com/15204313/131919989-09e2c4a6-524c-449a-870d-d91054923234.gif)

Another one recorded with gifcap

![recording](https://user-images.githubusercontent.com/15204313/131923066-fea1c370-f8c8-4e6f-8097-55e15dfa71aa.gif)

**Comment by meganrogge** [comment]
do you know when this started happening?

**Comment by RicaBenhossi** [comment]
No... I was using it inside a container because I need pyhton 3.7+ version and elementary 5.7 had python 3.6 by default. So, I was using the integrated terminal on container so far.
After upgrading elementary to version 6, that has python 3.8, I stop using container and though it was more productive for me to use external terminal and was when I found this issue.

**Comment by Tyriar** [comment]
I don't actually see the open in external terminal entry at all 🤔 

![image](https://user-images.githubusercontent.com/2193314/132345965-4989d8f8-ee3a-4756-a826-01e587554e95.png)

Adding to September to investigate

**Comment by RicaBenhossi** [comment]
Hum..... maybe is it because you're using VSCode on Windows and I'm using on Elementary os (based on Ubuntu 20.04)? I've tried on Windoes too but these options you post didn't show up for me.... May it is some configuration diferences.

**Comment by RicaBenhossi** [comment]
Sorry but this didn't looks like a diplicate of #132718. The #132718 is about **integrated** terminal and this is about open in external terminal.

**Comment by Tyriar** [comment]
Yeah that's the wrong link thanks. I swear this got fixed via a PR but can't find it 🤔

**Comment by meepo3927** [comment]
I got the same problem , already back to version 1.55

**Comment by meganrogge** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
